c10/core/QScheme.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+
+namespace c10 {
+
+/**
+ * QScheme is an enum that specifies the type of quantization. This has a one
+ * to one correspondence with Quantizer
+ * Please refer to ATen/core/Quantizer.h to see the Quantizers classes.
+ */
+enum class QScheme : uint8_t {
+  NO_QUANT,
+  PER_LAYER_AFFINE,
```

Comment:
ok, if global is a bad name, what about "uniform"? I find "per layer" a bit off, but maybe it's a standard terminology.

Reply:
Unfortunately uniform has some other meaning in quantization scheme.

Reply:
changing to PerTensor